<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateItem</class>
 <widget class="QDialog" name="UpdateItem">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Any Item</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
  background-color: #fbeee0;
  border: 2px solid #422800;
  border-radius: 30px;
  box-shadow: #422800 4px 4px 0 0;
  color: #422800;
  cursor: pointer;
  display: inline-block;
  font-weight: 600;
  font-size: 18px;
  padding: 0 18px;
  line-height: 50px;
  text-align: center;

}

QLineEdit{
  border: none;
  border: 2px solid #422800;
background-color: #fbeee0;
font-weight: 600;
  font-size: 18px;
}
QLabel{
font-weight: 600;
  font-size: 18px;
}
QComboBox{
background-color: #fbeee0;
  border: 2px solid #422800;
  border-radius: 30px;
  box-shadow: #422800 4px 4px 0 0;
  color: #422800;
  cursor: pointer;
  display: inline-block;
  font-weight: 600;
  font-size: 18px;
  padding: 0 18px;
  line-height: 50px;
  text-align: center;
}
QRadioButton{
background-color: #fbeee0;
font-weight: 600;
  font-size: 14px;
}
QGroupBox{
font-weight: 600;
  font-size: 14px;
}</string>
  </property>
  <widget class="QLineEdit" name="txtId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set Status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbStatus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Expired</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Available</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Returned</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spoiled</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Item ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtQty">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set Qty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
